Task 6/Task 6.4/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ru" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name = "viewport" content = "width = device-width, initial-scale = 1.0">
+    <title>Хостинг</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+  <body>
+    <section class = "mainSection">
+      <div class="section">
+        <img src="data-collection.png" alt="Cloud icon" class = "image"></img>
```

Comment:
* Not a cloud icon
* This can be implicitly styled (or class name could be less naive)